newIDE/app/src/UI/Dialog/index.js

```diff
@@ -86,6 +97,26 @@ type DialogContentStyle = {
   flexDirection?: 'row',
 };
 
+const findAndClickButton = (ref: {| current: ?HTMLElement |}): void => {
```

Comment:
the risk with this function is if we create a new fancy button in the dialog, the querySelector may not work anymore

Reply:
We don't add buttons everyday and we would only need to change a bit of logic/a css selector so I'm pretty confident it won't cause troubles before a long time